src/quote.mli

```diff
@@ -0,0 +1,9 @@
+open Import
```

Comment:
This mli needs documentation, otherwise it's not easy to understand what it does without looking at the implementation

Reply:
Yup, I will write some documentation today.